code/modules/mining/lavaland/megafauna_loot.dm

```diff
@@ -249,6 +249,8 @@
 	icon_state = "vial"
 
 /obj/item/mayhem/attack_self(mob/user)
+	if(tgui_alert(user, "Breaking the bottle will cause nearby crewmembers to go into a murderous frenzy. Be sure you know what you are doing","Break the bottle?",list("Break it!","DON'T")) != "Break it!")
+		return
```

Comment:
You need to check that the user is still holding the item, and that they are still in a valid state to break it, after the input
```dm
if(QDELETED(src) || loc != user || user.incapacitated)
	return
```

Reply:
Yeah, this makes sense